getting-started/nextjs.mdx

```diff
@@ -0,0 +1,275 @@
+---
+title: "NextJS Support"
+sidebarTitle: "NextJS Support"
+description:
+  "PropelAuth currently supports authentication in applications developed using the NextJS version 13.
+	This guide provides a simple way to integrate your Next.js application (either AppRouter or Pages) with PropelAuth."
+---
+
+<Tip>This guide is for getting started with using PropelAuth with NextJS version 13 
+with server side rendering. If you are only planning on using client side support, 
+please reference our React libraries instead. </Tip>
+
+## Installation
+
+First, be sure to install the following package in order to access these features.
+
+```bash
+npm install @propelauth/nextjs
+```
+
+## Setup
+
+Before you start, make sure you have a PropelAuth account. You can sign up for free at here.
+
+You'll need to set the following .env variables in your Next.js application:
+
+```NEXT_PUBLIC_AUTH_URL
+PROPELAUTH_API_KEY
+PROPELAUTH_VERIFIER_KEY
+PROPELAUTH_REDIRECT_URI
+```
+
+You can find the ```NEXT_PUBLIC_AUTH_URL```, ```PROPELAUTH_API_KEY```, and ```PROPELAUTH_VERIFIER_KEY``` variables for your 
+application in the PropelAuth Dashboard under Backend Integration.
+
+When you copy the ```PROPELAUTH_VERIFIER_KEY``` from the PropelAuth dashboard, it will automatically paste into 
+your .env file with line breaks. However, due to the way some systems interpret multiline environment 
+variables, you will need to edit the verifier key value to include '\n' instead of newline characters. For example:
+
+```PROPELAUTH_VERIFIER_KEY=-----BEGIN PUBLIC KEY-----\nMIIBIjANBgk...```
+
+For the ```PROPELAUTH_REDIRECT_URI``` variable, you need to add ```/api/auth/callback``` to the end of one of your 
+allowed frontend locations. So, for example, if you are developing in the test environment and 
+using http://localhost:3000, you would use http://localhost:3000/api/auth/callback
+
+### Set up routes
+
+In your `src/app/api/auth/[slug]` directory, create a file called `route.ts` with the following content:
+
+```tsx
+import {getRouteHandler, postRouteHandler} from "@/auth";
+
+export const GET = getRouteHandler
+export const POST = postRouteHandler
+```
+
+### Set up AuthProvider
+
+<Tabs>
+  <Tab title="Using AppRouter">
```

Comment:
It's probably best to just match Next.js' convention here and say App Router and Pages Router